components/optional/index.ts

```diff
@@ -0,0 +1,25 @@
+import { useBlockEditContext } from '@wordpress/block-editor';
+
+interface OptionalProps {
+	/**
+	 * The value to check for truthiness.
+	 */
+	value?: string | number | boolean;
+
+	/**
+	 * The children to render if the value is truthy.
+	 */
+	children: React.ReactNode;
+}
+
+export const Optional: React.FC<OptionalProps> = ({
+	value,
```

Comment:
Lets also move the `.defaultValue` to an inline declaration here

```suggestion
	value = '',
```